<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Main">
       <attribute name="title">
        <string>Библиариум</string>
       </attribute>
       <widget class="QLabel" name="description">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>500</y>
          <width>391</width>
          <height>261</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>421</width>
          <height>761</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="niga">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>311</width>
          <height>461</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../Desktop/2836943_detail.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="zaklad">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>725</x>
          <y>20</y>
          <width>20</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>non_fav.png</normaloff>non_fav.png</iconset>
        </property>
       </widget>
       <widget class="QListWidget" name="spisok">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>321</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>261</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Фильтры</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>241</width>
           <height>81</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Жанры:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="genre">
            <property name="currentText">
             <string>Все</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtCurrent</enum>
            </property>
            <item>
             <property name="text">
              <string>Все</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Авторы:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Страна:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="country">
            <item>
             <property name="text">
              <string>Все</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="autor">
            <item>
             <property name="text">
              <string>Все</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="podbor">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Подобрать</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Поиск по названию</string>
        </property>
        <widget class="QPushButton" name="lupa">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>20</y>
           <width>31</width>
           <height>28</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>20</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>102</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>but.png</normaloff>but.png</iconset>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>241</width>
           <height>22</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="stroka"/>
          </item>
         </layout>
        </widget>
       </widget>
       <zorder>listView</zorder>
       <zorder>description</zorder>
       <zorder>niga</zorder>
       <zorder>zaklad</zorder>
       <zorder>spisok</zorder>
       <zorder>groupBox</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
      <widget class="QWidget" name="Mark">
       <attribute name="title">
        <string>Закладки</string>
       </attribute>
       <widget class="QListWidget" name="spisok_fav">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="listView_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>421</width>
          <height>761</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nigg_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>311</width>
          <height>461</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../Desktop/2836943_detail.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="description2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>500</y>
          <width>391</width>
          <height>261</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="DB">
       <attribute name="title">
        <string>DB</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
